migrations/versions/221cf4ac610a_assosiating_a_list_of_books_to_user.py

```
@@ -0,0 +1,30 @@
+"""assosiating a list of books to user
+
+Revision ID: 221cf4ac610a
+Revises: caa1c8250ffb
+Create Date: 2024-12-26 17:25:30.066074
+
+"""
+from typing import Sequence, Union
+import sqlmodel
```

Ruff (0.8.2)

`sqlmodel` imported but unused

Remove unused import: `sqlmodel`

(F401)